javascript-practice/src/pages/dashboard.html

```diff
@@ -1,11 +1,53 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Dashboard</title>
-</head>
-<body>
-  
-</body>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../styles/main.css" />
+    <title>Dashboard</title>
+  </head>
+  <body>
+    <div class="container">
+      <header class="header-dashboard flex">
+        <div class="header-dashboard-name-box flex-column">
+          <div class="flex header-dashboard-hi-name-box">
+            <h2 class="header-dashboard-name">Hello,</h2>
+            <!-- this is fake data. I'll update it in the next task -->
```

Comment:
Comment is not correct. Use TODO: https://www.jetbrains.com/help/webstorm/using-todo.html#create_todo